lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -1671,7 +1628,7 @@ static LogicalResult verifyPartialConnectOp(PartialConnectOp partialConnect) {
            << ". Types are not weakly equivalent.";
 
   // Check that the flows make sense.
-  if (foldFlow(partialConnect.src()) == Flow::Sink) {
+  if (FlowKind::get(partialConnect.src()) == Flow::Sink) {
```

Comment:
possible simplification: FlowKind::isSink(val)?

Reply:
I like it, added for all three kinds of flow.